### PR TITLE
Wires final

### PR DESCRIPTION
**Pull request by ziofil**
sync `wires-docs-tests` with `wires-final` before merging it into `develop` and `protytpe-tn`

### CODE REVIEW

File: mrmustard/lab/wires.py [hunk 1]
```diff
@@ -0,0 +1,327 @@
+# Copyright 2023 Xanadu Quantum Technologies Inc.
```
Comment (SamFerracin) [inline mrmustard/lab/wires.py:1]: remove this file?

File: tests/test_lab/test_wires.py [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Tests for Wires class."""
+
+# pylint: disable=missing-function-docstring
+# pylint: disable=protected-access
+
+import pytest
```
Comment (SamFerracin) [inline tests/test_lab/test_wires.py:20]: remove this file?

**Review by SamFerracin — COMMENTED** [review]
just a couple of files to remove

**Comment by ziofil** [comment]
better to merge the other way around

**Pull request closed without merging**